### PR TITLE
Mscraven patch 1

### PR DESCRIPTION
**Pull request by mscraven**
Changes to deal with multiple delete forms on one page and add a small check to keep from adding a boolean when user hits enter on a nonautocomplete field. See my issue in the issue queue.  Hope this works out.  Thanks.

### CODE REVIEW

File: apachesolr_angularjs_search.module [hunk 1]
```diff
@@ -1175,10 +1175,23 @@ function apachesolr_angularjs_search_uri_exists($uri) {
...
 }
 
+ /**
+ * Change form ids for multiple delete forms.
+ */
+function apachesolr_angularjs_search_forms($form_id, $args) {
+  $forms = array();
+  if(preg_match('/^apachesolr_angularjs_search_delete_form_\d+/', $form_id)) {
+    $forms[$form_id] = array(
+                   'callback' => 'apachesolr_angularjs_search_delete_form_callback'
```
Comment (kporras07) [inline apachesolr_angularjs_search.module]: Could you please ensure you have only 2 spaces for indentation in this line and a final comma?

File: apachesolr_angularjs_search.module [hunk 2]
```diff
@@ -1188,6 +1201,10 @@ function apachesolr_angularjs_search_delete_form($form, $form_state, $sid, $dele
     '#type' => 'submit',
     '#value' => check_plain($delete_text),
   );
+  
+  $form['#submit'] = array (
+    'apachesolr_angularjs_search_delete_form_submit'
```
Comment (kporras07) [inline apachesolr_angularjs_search.module]: Could you please add a final comma?

File: src/components/booleansPopup/booleansPopup.directive.js [hunk 3]
```diff
@@ -116,7 +116,7 @@
               jQuery('.advanced-search--field-autocomplete-operator').parents('.ui-select-match-item').children('.ui-select-match-close').remove();
             }, 0);
           }
-          else {
+           else if ($event.explicitOriginalTarget === $event.target) {
```
Comment (kporras07) [inline src/components/booleansPopup/booleansPopup.directive.js]: Could you please remove the extra space at the beginning of the line?

File: src/components/booleansPopup/booleansPopup.directive.js [hunk 4]
```diff
@@ -124,7 +124,7 @@
           }
           var selectionSum = operator.length + 2;
           var cursorPosition = selectionStart + selectionSum;
-          if (target) {
+           if (target && $event.explicitOriginalTarget === $event.target) {
```
Comment (kporras07) [inline src/components/booleansPopup/booleansPopup.directive.js]: Could you please remove the extra space at the beginning of the line?

File: views/apachesolr_angularjs_search_handler_field_delete_link.inc [hunk 5]
```diff
@@ -40,7 +40,7 @@ class apachesolr_angularjs_search_handler_field_delete_link extends views_handle
   function render($values) {
     $link_text = $this->options['link_text'];
     $sid = $values->sid;
-    $form = drupal_get_form('apachesolr_angularjs_search_delete_form', $sid, $link_text);
+     $form = drupal_get_form('apachesolr_angularjs_search_delete_form_'.$sid, $sid, $link_text);
```
Comment (kporras07) [inline views/apachesolr_angularjs_search_handler_field_delete_link.inc]: Could you please remove the extra space at the beginning of the line and add a space to both sides of the concatenation dot?

**Comment by kporras07** [comment]
Hi @mscraven it's great to receive a PR for this; much appreciated.
Could you please make the minor changes I asked in order to adhere to Drupal coding standards?

**Comment by mscraven** [comment]
OK - made changes. Hope it helps! Thanks for looking at my suggestions.

**Comment by kporras07** [comment]
Hi @mscraven; it's great to see you made the changes; code looks ok; I'll take a functional look to this ASAP (I hope to do it the next week). Huge thanks!

**Comment by kporras07** [comment]
This should fix https://github.com/ManatiCR/apachesolr_angularjs_search/issues/15

**Comment by kporras07** [comment]
@mscraven thanks for your contribution (and sorry for the big delay)

**Pull request merged**